content/developer/reference/extract_api.rst

```diff
@@ -1,416 +1,672 @@
-:code-column:
-
 ===========
 Extract API
 ===========
 
-Odoo provides a service allowing you to automate the processing of your invoices. The service scans your document using an Optical
-Character Recognition (OCR) engine and then uses AI-based algorithms to extract the fields of interest such as the total, the due date, or
-the invoice lines. More functional information can be found on the `demo page <https://www.odoo.com/page/invoice-automation>`_.
+Odoo provides a service to automate the processing of documents of type **invoices**, **expenses**
+or **resumes**.
 
-This service is a paid service. Each invoice processing will cost you one credit. Three different sized packs can be bought on `iap.odoo.com <https://iap.odoo.com/iap/in-app-services/259?sortby=date>`_.
+The service scans documents using an :abbr:`OCR (Optical Character Recognition)` engine and then
+uses :abbr:`AI(Artificial Intelligence)`-based algorithms to extract fields of interest such as the
+total, due date, or invoice lines for *invoices*, the total, date, payment reference for *expenses*,
+or the name, email, phone number for *resumes*.
 
-You can either use this service directly in the Odoo Accounting App or through the API. The Extract API which is detailed in the next section
-allows you to integrate our service directly into your own projects.
+This service is a paid service. Each document processing will cost you one credit. Three different
+sized packs can be bought on
```

Comment:
```suggestion
This service is a paid service. Each document processing will cost you one credit.
Credits can be bought on
```
In the case where we change the packs (remove one or add one), there won't be an inconsistency between this page and the IAP services page.